en/docs/learn/analytics/configuring-keystores-in-apim-analytics.md

```diff
@@ -1,175 +1,213 @@
       
     | Element                | Description                                                      |
     |------------------------|------------------------------------------------------------------|
-    | **keyStoreFile**       | Name of the keystore file. **The file must be located in `<API-M_ANALYTICS_HOME>/resources/security`** |
+    | **keyStoreFile**       | Name of the keystore file. **The file must be located in the `<API-M_ANALYTICS_HOME>/resources/security` directory.** |
     | **keyStorePassword**   | Keystore password                                                |
     | **certPass**           | Private key password                                             |
    
-3. **Securevault configuration**
+3. Update the Secure Vault configurations.
+   
+     1. Update the configurations in the `deployment.yaml` file.
+    
+         The Secure Vault configurations are used to store encrypted passwords that are mapped to aliases.
+
+          ```yaml
+          wso2.securevault:
+            secretRepository:
+              ...
+              parameters:
+                privateKeyAlias: wso2carbon
+                keystoreLocation: ${sys:carbon.home}/resources/security/securevault.jks
+                ... 
+          ```
+          The elements in the above configuration are described below:
+            
+          | Element                | Description                                                      |
+          |------------------------|------------------------------------------------------------------|
+          | **privateKeyAlias**    | Private key alias                                                |
+          | **keyStoreLocation**   | Name of the keystore file. **The file must be located in the `<API-M_ANALYTICS_HOME>/resources/security` directory.** |
 
-    This is used to store encrypted passwords that are mapped to aliases.
-    ```yaml
-       wso2.securevault:
-         secretRepository:
-           ...
-           parameters:
-             privateKeyAlias: wso2carbon
-             keystoreLocation: ${sys:carbon.home}/resources/security/securevault.jks
-             ... 
-    ```
-    The elements in the above configuration are described below:
+     2. Provide the keystore and private key password in the Base64 encoded format in the `<ANALYTICS_HOME>/conf/worker/master-keys.yaml` file.
+
+        ??? info "Click here for more info on the `master-keys.yaml` file"
+            The base64 encoded plaintext keystore password and the base64 encoded plaintext private password of the keystore used for Secure Vault is defined in the master-keys.yaml file. WSO2 Carbon Server reads this file to obtain these two passwords, so that it can decrypt the passwords in the `secrets.properties` file in order to start the server.
+
+4. Set the new keystore details when you change the keystore.
+
+      1. Navigate to the <ANALYTICS_HOME>/wso2/dashboard/bin/carbon.sh` file.
```

Comment:
```suggestion
      1. Navigate to the <ANALYTICS_HOME>/wso2/worker/bin/carbon.sh` file.
```

Reply:
Fixed in https://github.com/wso2/docs-apim/pull/1231/commits/f872500a49ef0c6223fab16659a66f2e133faa22

Reply:
Still, this change is not fixed.